src/main/java/controllers/FeedController.java

```diff
@@ -63,18 +63,22 @@ private void runBrowseFeed() {
      */
     public String generateDisplayedPosts(ArrayList<Post> posts) {
         this.postsActionMap = new HashMap<>();
-        StringBuilder postsString = new StringBuilder("Enter a post number for a detailed view of " +
+        StringBuilder postsString = new StringBuilder("Enter a post number (if there's any) for a detailed view of " +
```

Comment:
Don't we handle the case on line 69 where posts.size() == 0? If we are handling it, why do we need to say "if there's any"? Wouldn't there always be at least one post when this methods shows, or am I misunderstanding the code?

Reply:
The "posts" parameter of the method represents the list of displayed posts in a feed after being filtered, so in some cases, the variable posts may be an empty list of posts. Also, I will get rid of the "if there's any"